src/components/Card.js

```diff
@@ -28,15 +41,27 @@ export default class Card {
     });
   }
 
-  _handleLikeClick() {
-    this._likeButton.classList.toggle("card__like-button_active");
+  renderLikes(isLiked) {
+    this._isLiked = isLiked;
+    const likeButton = this._cardElement.querySelector(".card__like-button");
```

Comment:
`    const likeButton = this._cardElement.querySelector(".card__like-button");
`

You've found `this._likeButton` in line 20. Please, use it here

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.